scripts/setupdeps.sh

```diff
@@ -7,6 +7,11 @@ SUPPORTED_OS=('win64' 'maci64' 'maca64' 'glnxa64')
 DISTDIR="$(pwd)/dist/bin"
 mkdir -p $DISTDIR
 
+# Create plugin directory and copy plugin code
+PLUGINDIR="$(pwd)/dist/plugins/+ciplugins/+github"
+mkdir -p $PLUGINDIR
+cp -R plugins/+ciplugins/+github/*.m $PLUGINDIR/
```

Comment:
```suggestion
cp -R plugins $DISTDIR/
```

Reply:
I dont htink this will work as is we need to create plugins directory under DIST
<img width="296" alt="image" src="https://github.com/matlab-actions/run-build/assets/47204011/53fe828b-4ddb-4b53-bc09-a4a9d28c749e">

Reply:
Done